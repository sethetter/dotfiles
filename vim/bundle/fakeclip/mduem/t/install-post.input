#!/bin/bash

dir='t/,install-post'
makefile='t/,install-post.Makefile'

cat >"$makefile" <<END
TARGETS_POST_INSTALL = find

.PHONY: find
find:
	find '$dir' >&2

include Makefile
END

make --always-make -f "$makefile" >/dev/null \
  "INSTALLATION_DIR=$dir" \
  'TARGETS_STATIC=Makefile README' \
  install

rm -rf "$dir" "$makefile"

# __END__
