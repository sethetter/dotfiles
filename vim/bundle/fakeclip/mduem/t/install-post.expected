t/,install-post
t/,install-post/Makefile
t/,install-post/README
